app/human-detector-app/classes/Group.tsx

```diff
@@ -23,4 +23,11 @@ export default class Group {
   removeCameraFromGroup(camerasIndex: number) {
     this.cameras.splice(camerasIndex, 1);
   }
+  
+  getCameraFromId(cameraId: string) {
+    if (this.cameras.find((cam) => cam.cameraId === cameraId)) {
```

Comment:
Not that this really matters, but could you save off the result? Then could check if the variable is null and return it.

Reply:
We could also use the map changes that were made for the notif screen that are stored in the `UserContext` since it has all the cameras anyway.  You also have to worry about deleting from the map there when deleting a camera